### DISCUSSION TITLE
Quarkus 3.0.0.Beta1 released

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#32066] [Quarkus Blog/Website]
# Quarkus 3.0.0.Beta1 released

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-0-0-beta1-released/

**Comment by adampoplawski** [comment]
Hello
Does it mean you want to synchronize relase of Quarkus 3 and Reactive Hibernate 2?

**Reply by gsmet** [reply]
@adampoplawski not sure I understand your question. Hibernate Reactive 2 is a new major version of Hibernate Reactive based on Hibernate ORM 6.

**Reply by adampoplawski** [reply]
Hello
In what is new section there is "Hibernate Reactive has been reintroduced into Quarkus. Hibernate Reactive 2.0 is in Alpha and is based on Hibernate ORM 6.2". In Quarkus version 2 Hinernate Reactive is in version 1. So does it mean that Quarkus 3 will migrate hibernate reactive to version 2 in BOM?

**Reply by gsmet** [reply]
Yes. Quarkus 3 will enforce Hibernate Reactive 2.
